scripts/parse-instructor-info.py

```diff
@@ -37,7 +37,7 @@
         ('Programming with Python', 'swcarpentry', 'python-novice-inflammation', 'swc/python'),
         ('Programming with R', 'swcarpentry', 'r-novice-inflammation', 'swc/r'),
         ('Programming with MATLAB', 'swcarpentry', 'matlab-novice-inflammation', 'swc/matlab'),
-        ('Data Organization in Spreadsheets', 'datacarpentry', 'excel-ecology', 'dc/spreadsheets'),
+        ('Data Organization in Spreadsheets', 'datacarpentry', 'excel-ecology', 'dc/spreadsheet'),
```

Comment:
We should use "spreadsheets" (the plural form).  What do we have to change elsewhere to make this consistent?